synapse/lib/webapp.py

```diff
@@ -102,14 +102,14 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config={}, srv_config={}):
```

Comment:
@vEpiphyte - yep, I originally kept the signature the same, passed in the server config inside settings, then popped it back out.